setup/deb-setup.sh

```diff
@@ -117,6 +117,8 @@ else
 fi
 
 printf '[*] Installing Gems... \n'
+output_line "sudo apt install libmysqlclient-dev" && print_success "${CLEAR_LINE}[+] Gems installed\n"
+output_line "gem install mysql2" && print_success "${CLEAR_LINE}[+] Gems installed\n"
```

Comment:
This shouldn't be necessary to do separately, as `mysql2` gem is part of the bundle specified in the Gemfile.

Reply:
Hi @ragesoss, thank you for the feedback. Mysql2 gem installation was a bit problematic with the Gemfile in debian when mysqlclient is not installed so including it was helpful. So should I remove both line 120 and 121 or just 121?